pages/index.jsx

```diff
@@ -4,6 +4,9 @@ import styles from '../styles/home.module.css'
 import { getTree } from '../api/apiPage'
 import Manual from '../components/Manual/Manual'
 
+// TODO Remove hard-code after back-end feature https://github.com/ekaterinburgdev/guides-api/issues/10
+const HIDDEN_MANUALS = ['typical', 'typical-2', 'street-name-plates']
```

Comment:
For const, ideally, add new folder, but solution is temporary you can be left like this

Reply:
Done